chain/x/zoracle/client/rest/query.go

```diff
@@ -13,93 +13,98 @@ import (
 	"github.com/bandprotocol/d3n/chain/x/zoracle/internal/types"
 )
 
+func newRequestQueryInfo(
+	ctx context.CLIContext, id string, queryRequest types.RequestInfo,
+) (RequestQueryInfo, error) {
+	var request RequestQueryInfo
+
+	request.OracleScriptID = queryRequest.Request.OracleScriptID
+	request.Calldata = queryRequest.Request.Calldata
+	request.RequestedValidators = queryRequest.Request.RequestedValidators
+	request.SufficientValidatorCount = queryRequest.Request.SufficientValidatorCount
+	request.ExpirationHeight = queryRequest.Request.ExpirationHeight
+	request.IsResolved = queryRequest.Request.IsResolved
+
+	request.Result = queryRequest.Result
+	if queryRequest.Result == nil {
```

Comment:
I think we can leave it as `nil`. People that call this function should know that if `isResolved` is false then result is `nil`.